### PR TITLE
To candle

### PR DESCRIPTION
**Pull request by gihanpanapitiya**
Candle compatible SWnet

### CODE REVIEW

File: SWnet.def [hunk 1]
```diff
@@ -0,0 +1,56 @@
+Bootstrap: docker
+From: pytorch/pytorch:1.12.0-cuda11.3-cudnn8-devel
+
+%labels
+	MANTAINER Gihan Panapitiya
+
+%setup
```
Comment (wilke) [inline SWnet.def:7]: Missing environment section

File: SWnet.def [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	chmod +x train_self_attn_gdsc.sh
+	chmod +x train_self_attn_ccle.sh
+	chmod +x train_multi_task.sh
+	chmod +x eval_self_attn_gdsc.sh
+	chmod +x eval_self_attn_ccle.sh
+	chmod +x train.sh
+	chmod +x infer.sh
+	chmod +x preprocess.sh
+
+	cp train_self_attn_gdsc.sh /usr/local/bin
```
Comment (wilke) [inline SWnet.def:45]: Why not cp *.sh /usr/local/bin

File: SWnet_CCLE_self-attention_evaluate.py
Comment (wilke) [inline SWnet_CCLE_self-attention_evaluate.py:1]: LGTM, but don't know what this file is supposed to do.

File: SWnet_CCLE_self-attention_train.py
Comment (wilke) [inline SWnet_CCLE_self-attention_train.py:1]: LGTM, but don't know what this file is supposed to do.

File: SWnet_GDSC_baseline_pytorch.py
Comment (wilke) [inline SWnet_GDSC_baseline_pytorch.py:1]: Did not check.

File: SWnet_GDSC_self-attention_evaluate.py
Comment (wilke) [inline SWnet_GDSC_self-attention_evaluate.py:1]: LGTM, but don't know what this file is supposed to do.

**Review by wilke — APPROVED** [review]
I prefer smaller commits, which are easier to review. I will accept this PR and run local tests to confirm correctness. This is just the first initial commit of the candlelized code.

File: SWnet.def (hunk 2 above)
Comment (gihanpanapitiya) [reply SWnet.def:45]: Yeah, will change it.

File: SWnet_CCLE_self-attention_evaluate.py
Comment (gihanpanapitiya) [reply SWnet_CCLE_self-attention_evaluate.py:1]: Yeah, this is a old file which I no longer use. Is there a way to remove such files from PR. I still want to keep them in my repository.

File: SWnet_GDSC_baseline_pytorch.py
Comment (gihanpanapitiya) [reply SWnet_GDSC_baseline_pytorch.py:1]: A file not currently in use.

File: SWnet_GDSC_self-attention_evaluate.py
Comment (gihanpanapitiya) [reply SWnet_GDSC_self-attention_evaluate.py:1]: A file not currently in use. Can remove from the PR.

File: SWnet_CCLE_self-attention_train.py
Comment (gihanpanapitiya) [reply SWnet_CCLE_self-attention_train.py:1]: A file not currently in use. Can remove from the PR.

**Pull request merged**